crates/bevy_render/src/lib.rs

```diff
@@ -190,8 +190,8 @@ pub mod main_graph {
 }
 
 #[derive(Resource)]
-struct FutureRendererResources(
-    Arc<
+pub struct FutureRendererResources(
+    pub  Arc<
```

Comment:
I know this is unrelated to this PR, but rather than a Mutex, I feel like this should be a OnceLock. Thoughts?

Reply:
While I understand how a OnceLock would make more sense here, to extract the renderer resources outside of the Arc reference, we need a struct with interior mutability since wgpu::Instance does not implement Clone.